src/screens/RegistrationScreen/DonorRegistrationScreen.tsx

```diff
@@ -125,7 +125,7 @@ export default () => {
 							<TouchableWithoutFeedback
 								onPress={pickImage}
 							>
-								<Icon name="image" style={styles.icon} />
+								<Icon name="image" size={24} />
```

Comment:
Should standard icon sizes (18, 24) be added as exported constants from the Icon folder?